esx_service/cli/vmdkops_admin.py

```diff
@@ -509,6 +510,59 @@ def policy_update(args):
         print 'Successfully updated policy: {0}'.format(args.name)
 
 
+def status(args):
+    print "Version: {0}".format(get_version())
+    (status, pid) = get_service_status()
+    print "Status: {0}".format(status)
+    if pid:
+        print "Pid: {0}".format(pid)
+        print "Port: {0}".format(get_listening_port(pid))
+    print "LogConfigFile: {0}".format(log_config.LOG_CONFIG_FILE)
+    print "LogFile: {0}".format(log_config.LOG_FILE)
+    print "LogLevel: {0}".format(log_config.get_log_level())
+
+
+VMDK_OPSD = '/etc/init.d/vmdk-opsd'
+PS = 'ps -c | grep '
+GREP_V_GREP = ' | grep -v grep'
+NOT_RUNNING_STATUS = ("Stopped", None)
+
+def get_service_status():
+    """
+    Determine whether the service is running and it's PID. Return the 2 tuple
+    containing a status string and PID. If the service is not running, PID is
+    None
+    """
+    try:
+        output = subprocess.check_output([VMDK_OPSD, "status"]).split()
+        if output[2] == "not":
+            return NOT_RUNNING_STATUS
+
+        pid = output[3].split("=")[1]
+        return ("Running", pid)
+    except subprocess.CalledProcessError:
+        return NOT_RUNNING_STATUS
+
+
+def get_listening_port(pid):
+    """ Return the configured port that the service is listening on """
+    try:
+        cmd = "{0}{1}{2}".format(PS, pid, GREP_V_GREP)
+        output = subprocess.check_output(cmd, shell=True).split()[6]
+        return output
+    except:
+        return NOT_AVAILABLE
+
```

Comment:
I agree that it would be great if there was an admin port to allow the cli to ask questions of the plugin server. I think that's an enhancement we should make in the future. @pdhamdhere @msterin @kerneltime Thoughts?